deploy/mysql/init.sql

```diff
@@ -31,9 +31,10 @@ CREATE TABLE `catalogs` (
 	  `bundle` longtext,
 	  `description` longtext,
 	  `icon_data` longtext,
-	  `user_id` bigint(20) unsigned DEFAULT NULL,
+	  `account_id` bigint(20) unsigned DEFAULT NULL,
 	  `type` int(10) unsigned DEFAULT NULL,
-	  PRIMARY KEY (`id`)
+	  PRIMARY KEY (`id`),
+	  UNIQUE KEY `accountid_name` (`account_id`, `name`)
```

Comment:
We have to promise the version upgrade by DB migration scripts from the coming version